scripts/blog_template.py

```diff
@@ -40,27 +74,48 @@ def fix_img_paths(data: pd.DataFrame) -> pd.Series:
     A function for fixing image paths generated by Microsoft forms
     """
 
-    assert 'image_file' in data.columns 
+    assert "image_file" in data.columns
+
+    return (
+        data["image_file"]
+        .fillna("")
+        .apply(
+            lambda x: ",".join(
+                [x.split("/")[-1].replace("%20", "_") for x in x.split(";")]
```

Comment:
This is exciting!
Apply an anonymous function to each row in the "image_file" columns that:
- joins all the items in a list into a comma separated string
- But the list we pass join is in fact created using a list comprehension where we take the string in the "image_file" column split it on a semi-colon to yield a list, for each item in this initial list split it again on a `/`, pick out the last item (which will be the filename) and with that string replace any occurrences of `%20` with `_`.
This returns a string separated with commas with just the filenames of each uploaded figure.